force-app/main/default/lwc/geFormWidget/geFormWidget.html

```diff
@@ -5,13 +5,11 @@
     <template if:true={isElevateTokenizeCard}>
         <!-- Do not render if gift has a schedule -->
```

Comment:
These two comment lines should be removed.

Reply:
totally missed that. thanks for calling those out. updated.